integration-tests/src/test/java/io/testerra/test/status/CheckTestStatusTest.java

```diff
@@ -95,13 +98,13 @@ public static Object[][] provideFinalTestResult() {
                 {"*** Stats: SuiteContexts:  2", "SuiteContext"},
                 {"*** Stats: TestContexts:   2", "TestContext"},
                 {"*** Stats: ClassContexts:  5", "ClassContext"},
-                {"*** Stats: MethodContexts: 43", "MethodContexts"},
-                {"*** Stats: Test Methods Count: 40 (30 relevant)", "Test methods"},
-                {"*** Stats: Failed: 7", "Failed tests"},
+                {"*** Stats: MethodContexts: 47", "MethodContexts"},
+                {"*** Stats: Test Methods Count: 44 (34 relevant)", "Test methods"},
+                {"*** Stats: Failed: 8", "Failed tests"},
```

Comment:
Sanity Check: In lines 56ff. we added _two_ failing and _two_ passing tests, yet down here, we add _one_ failing and _three_ passing tests. ... Is this fine?